ts/src/pro/bitopro.ts

```diff
@@ -261,6 +261,107 @@ export default class bitopro extends bitoproRest {
+        }
+        return this.filterBySymbolSinceLimit (orders, symbol, since, limit, true);
+    }
+
+    handleOrders (client, message) {
+        //
+        //     {
+        //         "event": "ACTIVE_ORDERS",
+        //         "timestamp": 1639552073346,
+        //         "datetime": "2021-12-15T07:07:53.346Z",
+        //         "data": {
+        //             "sol_usdt": [
+        //                 {
+        //                     "id": "8917255503",
+        //                     "pair": "sol_usdt",
+        //                     "price": "107",
+        //                     "avgExecutionPrice": "0",
+        //                     "action": "SELL",
+        //                     "type": "LIMIT",
+        //                     "timestamp": 1639386803663,
+        //                     "updatedTimestamp": 1639386803663,
+        //                     "createdTimestamp": 1639386803663,
+        //                     "status": 0,
+        //                     "originalAmount": "0.02",
+        //                     "remainingAmount": "0.02",
+        //                     "executedAmount": "0",
+        //                     "fee": "0",
+        //                     "feeSymbol": "usdt",
+        //                     "bitoFee": "0",
+        //                     "total": "0",
+        //                     "seq": "SOLUSDT3273528249",
+        //                     "timeInForce": "GTC"
+        //                 }
+        //             ],
+        //             "usdc_twd": [
+        //                 {
+        //                     "id": "3452766477",
+        //                     "pair": "usdc_twd",
+        //                     "price": "10",
+        //                     "avgExecutionPrice": "0",
+        //                     "action": "BUY",
+        //                     "type": "LIMIT",
+        //                     "timestamp": 1638258713957,
+        //                     "updatedTimestamp": 1639386803663,
+        //                     "createdTimestamp": 1639386803663,
+        //                     "status": 0,
+        //                     "originalAmount": "0.01",
+        //                     "remainingAmount": "0.01",
+        //                     "executedAmount": "0",
+        //                     "fee": "0",
+        //                     "feeSymbol": "usdc",
+        //                     "bitoFee": "0",
+        //                     "total": "0",
+        //                     "seq": "USDCTWD2310459465"
+        //                     "timeInForce": "GTC"
+        //                 }
+        //             ]
+        //         }
+        //     }
+        //
+        const event = this.safeString (message, 'event');
+        const data = this.safeValue (message, 'data');
+        const symbols = Object.keys (data);
+        const limit = this.safeInteger (this.options, 'ordersLimit', 1000);
+        const orders = new ArrayCacheBySymbolById (limit);
```

Comment:
@sc0Vu should we initialize the cache every time we get an update?

Reply:
@sc0Vu can you check this comment?

Reply:
@carlosmiei they only push opened order for that topic. I still think how to cache order and update the status.
How do you think that we don't cache order for now?

Reply:
I think keeping the cache still makes sense even if it's for open orders.

Also according to the docs they do pass the status? (https://github.com/bitoex/bitopro-offical-api-docs/blob/master/v3-1/rest-1/rest.md#order-status)

Reply:
@pcriadoperez yes, but they only return open orders. If the order is cancelled, they don't return that order.

Reply:
@sc0Vu so it returns for new, partially matched and closed orders but not cancelled orders?

Reply:
@pcriadoperez yes